webview/src/experiments/components/table/TableHeader.tsx

```diff
@@ -184,9 +191,16 @@ const TableHeaderCellContents: React.FC<{
         onDrop={onDrop}
       />
       {canResize && (
+        /* eslint-disable jsx-a11y/no-static-element-interactions */
         <div
           {...column.getResizerProps()}
-          onMouseEnter={() => setMenuSuppressed(true)}
+          onMouseDownCapture={() => {
```

Comment:
Are you using `onMouseDownCapture` just to stop the event from bubbling up? I have no opinion on this, I'm just curious.

Reply:
`column.getResizerProps` uses the `onMouseDown` event so I just used the `onMouseDownCapture` so I didn't have to overwrite the event handler on `mousedown`. 

My original plan was to just use a wrapper function around the `column.getResizerProp` function but I was getting some typescript error that I couldn't make sense of. For now, I just used the `capture` version of the function since it appeared to work correctly.

Reply:
That's actually a smart idea.